c10/util/string_view.h

```diff
@@ -591,6 +591,20 @@ constexpr inline void swap(
 
 using string_view = basic_string_view<char>;
 
+constexpr bool string_view_starts_with(
```

Comment:
For C++20, these are implemented as methods on string_view, we should add a [olyfill to future proof it (and for folks who are building PyTorch with C++20 right now.

Reply:
I don't know how many years it will take to fully support C++20...

Reply:
At least leave a comment then lol

Reply:
> At least leave a comment then lol

OK, but in a later PR to avoid running the jobs again..